### DISCUSSION TITLE
Faces not getting detected

### DISCUSSION THREAD

**Discussion by marcelklehr** [nextcloud/recognize#865] [General]
Some people have reported their faces not getting detected as such. Sometimes even on portrait shots.

My suspicion is that the face detection quality in WASM mode is subpar, but I haven't confirmed this.

**Comment by klic** [comment]
Any chance help getting more insights? I do have a few non detected faces, and these are portrait shots. Running on a Raspbi 4 on fedora, NC is natively installed

**Reply by marcelklehr** [reply]
Raspi is ARM, afaik, so you're also running in WASM mode, correct?

**Comment by klic** [comment]
Yes, sorry for not being more specific.

It's arm64, and I'm running in WASM mode
